spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/sql/CommandsHarvester.scala

```diff
@@ -202,7 +202,7 @@ object CommandsHarvester extends AtlasEntityUtils with Logging {
 
   def prepareEntity(tableIdentifier: TableIdentifier): AtlasEntityWithDependencies = {
     val tableName = tableIdentifier.table
-    val dbName = tableIdentifier.database.getOrElse("default")
+    val dbName = tableIdentifier.database.getOrElse(SparkUtils.getCurrentDatabase)
```

Comment:
Do we also need to worry about the case sensitivity? In the spark code I see it converts the DB name to lowercase based on the setting ([here](https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala#L130)). Just wondering if something similar needs to be done.

Reply:
Good point. Spark also has similar code for what we're doing here, so we could follow with it:

https://github.com/apache/spark/blob/6a317c8f014557dfd60931bd1eac3f545520d939/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala#L403-L405

In fact, `currentDb` should have applied `formatDatabaseName` before assigning, so effectively we can just use it,

https://github.com/apache/spark/blob/6a317c8f014557dfd60931bd1eac3f545520d939/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala#L254-L267

but Spark calls `formatDatabaseName` anyway so ideal to follow their pattern.

Reply:
You are still planning to make this change for both tableName and dbName, right? And I am guessing you may have already made sure that everywhere we are using SparkUtils methods to find dbName and tableName, but I just wanted to call that out.

Reply:
Yes I found all accessing `tableIdentifier` and changed them via 1c070f3.

Reply:
I found this change is not outdated while talking with @ankuriitg - sorry I'll make the change quickly.